### PR TITLE
Merged with kickass tooltips plugin.

### PR DESCRIPTION
**Pull request by brushesm**
Clean repository with merged kickass tooltips.

### CODE REVIEW

**Comment by Swoffa** [comment]
Thanks!! Will check as soon as I can.

File: Main.sublime-menu [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                                    "file": "${packages}/SublimeKickAssemblerC64/KickAssembler (C64).sublime-settings",
+                                },
+                                "caption": "Settings – Default"
+                            },
+                            {
+                                "command": "open_file",
+                                "args": {"file": "${packages}/User/KickAssembler (C64).sublime-settings"},
+                                "caption": "Settings – User"
+                            },
+                            { "caption": "-" },
```
Comment (Swoffa) [inline Main.sublime-menu]: I suggest removal of line 34-46, or I will do it later. I will merge `SublimeKickAssemblerC64/Preferences.sublime-settings` into `/SublimeKickAssemblerC64/KickAssembler (C64).sublime-settings`, don´t know why I put stuff in different files.

File: KickassTooltips.sublime-settings [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Swoffa) [inline KickassTooltips.sublime-settings]: I suggest merging all settings into one file `SublimeKickAssemblerC64/KickAssembler (C64).sublime-settings`. This means that for the user only one package exist, not 2. Good or bad idea?

File: Main.sublime-menu (hunk 1 above)
Comment (Swoffa) [reply Main.sublime-menu]: If good idea, can also be done later, after merging PR.

File: README-KickassTooltips.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
+KickassTooltips
```
Comment (Swoffa) [inline README-KickassTooltips.rst]: Suggest also to merge readme-files, but maybe later. :)

**Comment by Swoffa** [comment]
Functionality seems to work great!
Review comments and then we can merge.

**Comment by Swoffa** [comment]
I made a basic `opcodes.json`-file (no descriptions), can merge it later.

File: helpdb/sid-registers.json [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+        "url": ""
+    },
+
+    "d406": {
+        "descr": "Voice #1 Sustain volume and Release length. Write-only",
+        "name": "",
+        "url": ""
+    },
+
+    "d407": {
```
Comment (Swoffa) [inline helpdb/sid-registers.json]: Lots of sprite stuff in the SID-registers ;)

**Comment by og2t** [comment]
@brushesm Nice work!

File: Main.sublime-menu (hunk 1 above)
Comment (Swoffa) [reply Main.sublime-menu]: Never mind.

File: KickassTooltips.sublime-settings (hunk 2 above)
Comment (Swoffa) [reply KickassTooltips.sublime-settings]: Not a good idea.

File: Main.sublime-menu [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    {
+        "id": "preferences",
+        "children":
+        [
+            {
+                "id": "package-settings",
+                "children":
+                [
+                    {
+                        "caption": "SublimeKickAssemblerC64",
```
Comment (Swoffa) [inline Main.sublime-menu]: I included settings/preferences in menu for SublimeKickAssemblerC64 in master, but using, the `edit_settings`-command. You need to resolve some conflicts, sorry for that.

Comment (brushesm) [reply Main.sublime-menu]: Sorry for being out :) I have to learn the github workflow now :)

Comment (brushesm) [reply Main.sublime-menu]: ok. I think i resolved the conflict now.

File: helpdb/sid-registers.json (hunk 4 above)
Comment (brushesm) [reply helpdb/sid-registers.json]: Pushed updated version of the helpfile. I don't know if it'll automatically update the pull request. Reopen and kick me if not!

**Comment by Swoffa** [comment]
Finally merged! :)

**Pull request merged**